src/Nethermind/Nethermind.Runner/Program.cs

```diff
@@ -112,7 +112,18 @@ private static void Run(string[] args)
     {
         _logger.Info("Nethermind starting initialization.");
         _logger.Info($"Client version: {ProductInfo.ClientId}");
-
+        try{
+            string duplicateArgument = CheckForDuplicateArgs(args); 
+            if(!string.IsNullOrEmpty(duplicateArgument)){
+                Environment.ExitCode = ExitCodes.DuplicateArguments; 
+                throw new Exception($"Duplicate arguments passed - {duplicateArgument}.");;
+            }
+        }
+        catch(Exception e){
+            _logger.Error($"Failed due to duplicate arguments passed while execution - ", e);
+            _appClosed.Wait();
+            throw;
+        }
```

Comment:
Why the try->throw->catch? Simpler would be to just do log.Error, appClosed, return

Reply:
Did the suggested changes. Now the response to run command - `dotnet run --JsonRPC.EngineHost 0.0.0.0 --JsonRPC.EngineHost 0.0.0.0` looks like this - 

<img width="884" alt="image" src="https://github.com/NethermindEth/nethermind/assets/54079322/b2c5c721-d3f3-425c-95c6-fd36d93819bd">